.storybook/preview-head.html

```diff
@@ -7,5 +7,6 @@
     --mb-color-focus: #cbe2f7;
     --mb-color-bg-dark: #93a1ab;
     --mb-color-bg-light: #f9fbfc;
+    --mb-color-filter: #7172ad;
```

Comment:
should we also have `danger` and `error` here, as I think we have SmartScalar which uses `danger` for negative values (which I have storybook / Loki tests for)